packages/contracts/test/exchange/fill_order.ts

```diff
@@ -2,7 +2,10 @@ import { BlockchainLifecycle } from '@0xproject/dev-utils';
 import * as _ from 'lodash';
 
 import { chaiSetup } from '../utils/chai_setup';
-import { CoreCombinatorialUtils, coreCombinatorialUtilsFactoryAsync } from '../utils/core_combinatorial_utils';
+import {
```

Comment:
@fabioberger I renamed this file and some of the identifiers in it because they seem more focused on `fillOrder`. Does this sound right to you?

Reply:
Yep, makes sense. Initially I thought I'd get to more then fillOrder, but that didn't happen.